docs/10_domains/60_electrochemistry.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Electrochemistry"
+slug: "/electro_chemistry"
+---
+
+import { LbeChip } from "@site/src/components/lbe/LbeElements.js";
+import ElnFinder from "@site/src/components/eln/ElnFinder.js";
+
+<LbeChip title="physical chemistry" />
+<LbeChip title="electrochemistry" />
+
+## Introduction
+
+Electrochemistry plays a significant role in very different research areas, such as organic chemistry, analytical chemistry, life sciences, engineering and material science.
+Research is therefore often highly interdisciplinary, and the number of techniques and measurement procedures available are vast.
+
+## Data Types
+
+Conceptually, the main data type in electrochemistry consists of time series data containing fundamental electrical parameters such as voltage (potential)
+and current (current density), including the measurement time.
+For AC methods also the frequency, the impedance and the phase angle are relevant.
+These data are most simply stored as CSV, but are often only available in non-standard file formats.
```

Comment:
One could mention that a file standard for electrochemical data does not exist. Maybe some domains do have that, such as in electrosynthesis maybe JCAMP-DX?